test/e2e-common.sh

```diff
@@ -353,6 +352,19 @@ function install_knative_serving_standard() {
     fi
   fi
 
+  # Well, installing Knative with Ambassador is a sequential process, i.e. Knative serving needs to be installed before
+  # Ambassador. This is because Ambassador only enables Knative support when ingresses.networking.internal.knative.dev
+  # or clusteringresses.networking.internal.knative.dev CRDs are present, which is not the case if Ambassador is
```

Comment:
We can completely drop clusteringresses at this point. Let's not mention them here anymore?

Reply:
Fixed :+1: